components/ft-concept-button/main.scss

```
@@ -0,0 +1,239 @@
+
+	// stylelint-disable order/order
+	$theme-map: null !global;
+}
+
+@mixin _ftConceptButtonGetIcon($name, $color) {
+	@include oIconsContent(
+		$icon-name: $name,
+		$size: 10,
+		$color: $color,
+		$iconset-version: 1
+	);
+	content: '';
+}
+
+@mixin _ftConceptButtonPlusIcon($color) {
+	@include _ftConceptButtonGetIcon('plus', $color);
+	background-size: 25px;
+	margin: 0 6px -1px 0;
+}
+
+@mixin _ftConceptButtonTickIcon($color) {
+	@include _ftConceptButtonGetIcon('tick', $color);
+	background-size: 21px;
+}
+
+@mixin _ftConceptButtonIcon($theme: standard) {
+	@include _ftConceptButtonWithTheme($theme) {
+		&::before {
+			@include _ftConceptButtonPlusIcon(
+				_ftConceptButtonGetThemeColor(background)
+			);
+		}
+
+		&[aria-pressed='true'] {
+			&::before {
+				@include _ftConceptButtonTickIcon(_ftConceptButtonGetThemeColor(text));
+			}
+		}
+
+		&[disabled],
+		&[disabled]:hover {
+			background: transparent;
+
+			&::before {
+				@include _ftConceptButtonPlusIcon(
+					_ftConceptButtonGetThemeColor(disabled)
+				);
+				opacity: 0.5;
+			}
+
+			&[aria-pressed='true']::before {
+				@include _ftConceptButtonTickIcon(
+					_ftConceptButtonGetThemeColor(disabled)
+				);
+			}
+		}
+	}
+}
+
+@mixin _ftConceptButtonFocusOutlineColor($focus-color) {
+	// Apply :focus styles as a fallback
+	// These styles will be applied to all browsers that don't use the polyfill, this includes browsers which support the feature natively.
+	body:not(.js-focus-visible) &,
+	html:not(.js-focus-visible) & {
+		// Standardise focus styles.
+		&:focus {
+			outline: 2px solid $focus-color;
+		}
+	}
+
+	// When the focus-visible polyfill is applied `.js-focus-visible` is added to the html dom node
+	// (the body node in v4 of the 3rd party polyfill)
+
+	// stylelint-disable-next-line selector-no-qualifying-type
+	body.js-focus-visible &, // stylelint-disable-next-line selector-no-qualifying-type
+	html.js-focus-visible & {
+		// Standardise focus styles.
+		// stylelint-disable-next-line selector-no-qualifying-type
+		&.focus-visible {
+			outline: 2px solid $focus-color;
+		}
+		// Disable browser default focus style.
+		// stylelint-disable-next-line selector-no-qualifying-type
+		&:focus:not(.focus-visible) {
+			outline: 0;
+		}
+	}
+
+	// These styles will be ignored by browsers which do not recognise the :focus-visible selector (as per the third bullet point in https://www.w3.org/TR/selectors-3/#Conformance)
+	// If a browser supports :focus-visible we unset the :focus styles that were applied above
+	// (within the html:not(.js-focus-visible) block).
+	&:focus-visible,
+	body:not(.js-focus-visible) &:focus,
+	html:not(.js-focus-visible) &:focus {
+		outline: unset;
+	}
+
+	// Styles given :focus-visible support. Extra selectors needed to match
+	// previous `:focus` selector specificity.
+	body:not(.js-focus-visible) &:focus-visible,
+	html:not(.js-focus-visible) &:focus-visible,
+	&:focus-visible {
+		outline: 2px solid $focus-color;
+	}
```

I think this outline is [handled by o-normalise](https://github.com/Financial-Times/origami/blob/a4d4b745da06235623b03293f1ac1de1ca65116d/components/o-normalise/src/scss/_mixins.scss#L110) and all we need to do here is change the colour?
```suggestion
	&:focus,
	&.focus-visible {
		outline-color: $focus-color;
	}
```